src/config/dynamic-theme-fixes.config

```diff
@@ -11495,6 +11495,20 @@ INVERT
 
 ================================
 
+osu.edu
+ohio-state.edu
+
+CSS
+#osu_navbar {
+    background-image: linear-gradient(rgb(40, 43, 45) 0%, rgb(63, 69, 71) 100%);
+    border-bottom: 5px solid rgb(187, 0, 0);
+}
+body.sidebar-layout #page #content, #page #content {
```

Comment:
Where can I find these elements?

Reply:
I originally found it on these pages: [osu.edu/findpeople](https://www.osu.edu/findpeople/) and the [2FA Form](https://buckeyelink.osu.edu/?login=true) (redirects to it).
I just recently came across the same styling change on the [sitemap](https://www.osu.edu/a-z-site-list.html), but I'm sure there's more pages I'm missing.
![image](https://user-images.githubusercontent.com/16161144/151528233-df542c16-7a93-43b0-9ec5-fd5c256a5c05.png) (taken from sitemap)
Normally barely visible, the background image becomes much more noticeable when inverted.

I don't have a fix that'll work across most/all other pages.